src/System.Private.CoreLib/src/System/RtType.cs

```diff
@@ -4023,10 +3874,9 @@ public override MemberInfo[] GetDefaultMembers()
 
                 if ((bindingFlags & BindingFlags.PutRefDispProperty) != 0 && (bindingFlags & ClassicBindingMask & ~BindingFlags.PutRefDispProperty) != 0)
                     throw new ArgumentException(SR.Arg_COMPropSetPut, nameof(bindingFlags));
-                #endregion
 
                 {
-                    #region Non-TransparentProxy case
+                    // Non-TransparentProxy case
```

Comment:
This scope & comment can be deleted. (TransparentProxy refers to remoting that does not exist in .NET Core.)